docs/docs/zh/guide/howto/general-setting.md

```diff
@@ -12,8 +12,14 @@ DolphinScheduler 支持两种类型的内置主题，包括 `Dark` 和 `Light`
 
 ## 时区
 
-DolphinScheduler 支持时区设置。默认时区基于您运行 DolphinScheduler 服务器的时区。如果你想要切换时区，可以点击 [语言](#语言) 按钮右侧的时区按钮，
-然后点击 `请选择时区` 进行时区选择。当切换完成后，所有与时间相关的组件都将更改。
+DolphinScheduler 支持时区设置。
+
+服务时区
+
+使用脚本 `bin/dolphinshceduler_daemon.sh`启动服务， 服务的默认时区为UTC， 可以在`bin/env/dolphinscheduler_env.sh`中进行修改， 如`export SPRING_JACKSON_TIME_ZONE=${SPRING_JACKSON_TIME_ZONE:-Asia/Shanghai}`。<br>
```

Comment:
I think we should put a url of all the time zone or list all the time zone options here. WDYT? @caishunfeng

Reply:
It has so many time zones that it might be too long to here

Reply:
@caishunfeng What i mean is putting a link to the website that containing all time zones options.

Reply:
Good idea, what about the official url of java 8: 
https://docs.oracle.com/javase/8/docs/api/java/util/TimeZone.html#getAvailableIDs--

Reply:
I think we should give the results of ```TZ database name``` directly to the users instead of letting users to learn how to use it. ```https://en.wikipedia.org/wiki/List_of_tz_database_time_zones```. @caishunfeng

Reply:
>https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

I test some time zone and it's OK, I will add it into doc, thanks for your good suggestion.